### PR TITLE
Added Branding Page

### PR DESCRIPTION
**Pull request by AnAussiePilot**
Added HTML & CSS to Brandng

### CODE REVIEW

**Comment by scoopscode** [comment]
What is your Discord? (If you are in our Discord server)

**Comment by Megabrains44** [comment]
Looks good, maybe I'll add some of the previous styling on the other pages to branding.html

**Comment by AnAussiePilot** [comment]
mad

File: Branding.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <a href="https://www.youtube.com/channel/UCLwmhnbgYkvLHCKDCsFkTJw">YOUTUBE</a>
+            <a href="https://pinpointsimulationsform.herokuapp.com">APPLY</a>
+         </div>
+        </div>
+    </div>
+    <div class="copy6"><footer >BRANDING INFORMATION</footer></div>
+
+    <h2 class="about">Terms of Branding</h2>
+    <div class="info">
+      <p>If you would like to use our logo or images, and provide credit, you must follow the following conditions</p>
```
Comment (scoopscode) [inline Branding.html:34]: When I preview the site, all the text seems a bit off center. Making a new info class with different centering may work or you can modify this info class.
image

File: Branding.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      <p>1. Not obstruct the logo in any shape or form</p>
+      <p>2. Use only the provided colors (Hex: #FFFFFF & #C2272F)</p>
+      <p>3. Do not modify the logo.</p>
+      <p>4. Use transparent logo when needed, otherwise, please use the black background version.</p>
+      <p>5. We use the Brandon Grotesque font. It is a paid font, you may accomodate any modern font.</p>
+      <p>If you agree to these conditions, you may use the PPS Branding.</p>
+      <p>Find Logo in Transparent and Black Background Format at the Following Link : <a href="https://github.com/Pinpoint-Simulations/branding" >https://github.com/Pinpoint-Simulations/branding </a></p>
```
Comment (scoopscode) [inline Branding.html:41]:
```suggestion
      <p>Need help finding the logos? Follow this<a href="https://github.com/Pinpoint-Simulations/branding" target="_blank">link.</a></p>
```

**Review by scoopscode — CHANGES_REQUESTED** [review]

File: Branding.html (hunk 1 above)
Comment (AnAussiePilot) [reply Branding.html:34]: Aight. Will do

**Comment by Spherrrical** [comment]
How were you able to preview the site? I would like to see for myself then request any needed changes.

**Comment by AnAussiePilot** [comment]
Download the Files and run them in your browser

**Comment by Spherrrical** [comment]
Try to add CSS to the link to make it white and not blue. I'll download the files and see what I can do with it.

**Comment by scoopscode** [comment]
Can we get an update on this?

**Comment by scoopscode** [comment]
Closing due to inactivity

**Pull request closed without merging**